packages/perseus/src/util/parse-perseus-json/perseus-parsers/numeric-input-widget.ts

```diff
@@ -37,7 +38,10 @@ export const parseNumericInputWidget: Parser<NumericInputWidget> = parseWidget(
                 answerForms: optional(array(parseMathFormat)),
                 strict: boolean,
                 maxError: optional(nullable(number)),
-                simplify: optional(nullable(string)),
+                // TODO(benchristel): simplify should never be `true`, but we
+                // have some content where it is anyway. If we ever backfill
+                // the data, we should simplify `simplify`.
+                simplify: optional(nullable(pipeParsers(union(string).or(constant(true)).parser).then(convert(String)).parser)),
```

Comment:
Would you mind adding this note and an example to this doc? https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/3524690040/Content+Issues+for+Backfill